client/src/modules/profiles/graphql/queries.ts

```diff
@@ -11,9 +11,11 @@ export const userProfileQuery = gql`
       instance_role {
         name
       }
-      admined_chapters {
-        id
-        name
+      user_chapters {
+        chapter {
+          id
+          name
+        }
```

Comment:
We should provide them with event and ban info, too. It's data we store about them, after all.

Reply:
Added what I think user is interested in 👍